src/main/front/src/components/NodeGroupPage/MarkdownViwer.tsx

```
@@ -0,0 +1,41 @@
+
+const MarkdownViewer: React.FC<MarkdownViewerProps> = ({ content }) => {
+  return (
+    <Box
+      p={2}
+      //   bgcolor="#fdfdfd"
+      border="1px solid #ccc"
+      borderRadius={2}
+      sx={{ overflow: "auto", maxHeight: "100%" }}
+    >
+      <ReactMarkdown
+        children={content}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)